packages/docs-gesture-handler/docs/gestures/use-long-press-gesture.mdx

```diff
@@ -0,0 +1,175 @@
+---
+id: use-long-press-gesture
+title: Long press gesture
+sidebar_label: Long press gesture
+sidebar_position: 3
+---
+
+import { vanishOnMobile, appearOnMobile, webContainer } from '@site/src/utils/getGestureStyles';
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+import LongPressGestureBasic from '@site/static/examples/LongPressGestureBasic';
+import LongPressGestureBasicSrc from '!!raw-loader!@site/static/examples/LongPressGestureBasic';
+
+<div className={webContainer}>
+  <div className={vanishOnMobile} style={{ display: 'flex', justifyContent: 'center', maxWidth: 360 }}>
+    <video playsInline autoPlay muted loop style={{maxWidth: 360}}>
+      <source src={useBaseUrl("/video/longpress.mp4")} type="video/mp4"/>
```

Comment:
The video is on a different example 😢. It doesn't show anything on `onBegin`, so the square just changes color randomly.